app/assets/javascripts/sitewide/iframe_modal.js

```diff
@@ -13,6 +13,12 @@ $(document).on('turbolinks:load', function() {
     iFrameModal.dispatchEvent(new Event('shown'));
   };
 
+  iFrameModalFrame.addEventListener('load', () => {
+    iFrameModalFrame.contentWindow.document.querySelector('.close').addEventListener('click', function() {
```

Comment:
Good idea, but maybe the class should be a bit more specific in its naming, something like `.iframe-close`, to avoid potential conflicts from within the modal.